src/OSPSuite.SimModelNative/include/SimModel/Species.h

```diff
@@ -38,6 +38,7 @@ class Species :
 
 	Formula* createJacobianFor(const int index);
 	Formula* jacobianFor(const int index, const int mode, std::map<int, Formula*>& map);
+	Formula* _constantNullFormula;
```

Comment:
can we rename it to _constant**Zero**Formula?
Because when in C++, when I see _constantNullFormula, my first association is "`const NULL`" :)